http/whiteboard_http_server.go

```diff
@@ -0,0 +1,82 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/pivotal-sydney/whiteboardbot/app"
+	"io"
+	"net/http"
+	"os"
+)
+
+const (
+	DEFAULT_PORT = "9000"
+)
+
+type WhiteboardHttpServer struct {
+	SlackClient SlackClient
+	Whiteboard  Whiteboard
+}
+
+func (server WhiteboardHttpServer) Run() {
+	http.HandleFunc("/", server.NewHandleRequest())
+
+	if err := http.ListenAndServe(":"+server.getHealthCheckPort(), nil); err != nil {
+		fmt.Printf("ListenAndServe: %v\n", err)
+	}
+}
+
+func (server WhiteboardHttpServer) getHealthCheckPort() (port string) {
+	if port = os.Getenv("PORT"); len(port) == 0 {
+		fmt.Printf("Warning, PORT not set. Defaulting to %+v\n", DEFAULT_PORT)
+		port = DEFAULT_PORT
+	}
+	return
+}
+
+func (server WhiteboardHttpServer) NewHandleRequest() http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		req.ParseForm()
+
+		cmdArgs := req.FormValue("text")
+		token := req.FormValue("token")
+
+		if "" == os.Getenv("SLACK_TOKEN") || token != os.Getenv("SLACK_TOKEN") {
```

Comment:
This token is sent by Slack with every `POST` payload generated when someone executes a slash command. See the _How do commands work?_ section in the [Slack API docs](https://api.slack.com/slash-commands).